Makefile

```diff
@@ -23,5 +26,30 @@ all:
 	cp $(PATH_CRYPTO_TARGET)release/crypto_jni.dll $(PATH_JAVA_NATIVE_RESOURCES)
 endif
 
+MIN_ANDROID_VER := "29"
```

Comment:
Why did you pick 29 here?

Reply:
I think I copied it from somewhere, I changed it to 27, that's the minimum platform level available on the ubuntu 22.04 image :upside_down_face:

Reply:
Note that the current Android wallet has minSdkVersion 26, so this would be more restrictive than what we use there.

Reply:
Ahh, that is a good point, I'll test if it works on 26, and set it to that instead.